impala/hiveserver2.py

```diff
@@ -96,6 +97,9 @@ def cursor(self, user=None, configuration=None, convert_types=True,
             When `False`, timestamps and decimal values will not be converted
             to Python `datetime` and `Decimal` values. (These conversions are
             expensive.) Only applies when using HS2 protocol versions > 6.
+        convert_strings_to_unicode : bool, optional
+            When `False`, string values will not be converted to unicode. 
```

Comment:
The return type could be mentioned both for the True/False case.
+ It should be mentioned that this has effect only in Python3.